wavepool/admin.py

```diff
@@ -7,6 +7,21 @@ class NewsPostForm(forms.ModelForm):
     model = NewsPost
     fields = '__all__'
 
+    def clean(self, *args, **kwargs):
```

Comment:
Is the `clean` method the best place to put this logic? 

If for example a different part of the form does not validate, you have already made all the other stories `is_cover_story=False`. We probably only want to set them to not be the cover story, when we are sure that the this story passed validation and is going to be saved (if not saved already). 

It would make better sense for this logic to be in either the the `save` method of the `NewsPost` class, or the `save_model` method of the `NewsPostAdmin` class. The best place depends on whether or not we want it to always hold true whenever we save a `NewsPost`, or only when we are saving via the CMS.